go.sh

```diff
@@ -68,11 +68,10 @@ fi
 
 CT_NG=${SDK_NG_HOME}/bin/ct-ng
 
-
 mkdir -p build
 cd build
-export CT_PREFIX=`pwd`/output
-mkdir -p ${CT_PREFIX}/sources
+export OUTPUT_DIR=`pwd`/output
```

Comment:
Why was this change needed?

Reply:
This is to group all the Xtensa toolchains under one directory (see line 90-99 below). The intention is to have the same directory name as SoC name under `build/output/xtensa/${SOC}` in Zephyr cmake.